ultraScurve.py

```diff
@@ -57,8 +61,8 @@
 
 ohboard = getOHObject(options.slot,options.gtx,options.shelf,options.debug)
 
-SCURVE_MIN = 0
-SCURVE_MAX = 254
+SCURVE_MIN = options.scanmin
```

Comment:
Change these to be hardcoded to be the maximum range that works with the firmware so we don't have people not taking full scans.